<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NWindow</class>
 <widget class="QWidget" name="NWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackW">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>581</width>
     <height>671</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_32">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> РЛС</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tableRLS">
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_34">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Обстановка</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableObject">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_35">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Команды</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableCom">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Номер</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Тип</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Принята</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Обработана</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Доклад</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>Добавление РЛС</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>Да</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>Да</string>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string>Есть</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>Добавление объекта</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>Да</string>
         </property>
        </item>
        <item row="1" column="3">
         <property name="text">
          <string>Нет</string>
         </property>
        </item>
        <item row="1" column="4">
         <property name="text">
          <string>Нет</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>Изменение состояния</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>Да</string>
         </property>
        </item>
        <item row="2" column="3">
         <property name="text">
          <string>Нет</string>
         </property>
        </item>
        <item row="2" column="4">
         <property name="text">
          <string>Нет</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbInsertKom">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Сформировать команду</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_37">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Доклады</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableDoc">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pbClose">
          <property name="text">
           <string>Закрыть</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>651</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_38">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> РЛС</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_39">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Тип РЛС :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Номер РЛС: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="RLStableRLS">
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <column>
         <property name="text">
          <string>Сектор</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Статус</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dmax</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dmin</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Hmax</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Hmin</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_41">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Обстановка</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="RLStableObject">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="RLSpbInsObj">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Добавить объект</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_42">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Команды</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="RLStableCom">
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RLSpbInsertKom">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Обработать команду</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_43">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Доклады</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="RLStableDoc">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="RLSpbInsDoc">
          <property name="text">
           <string>Cформировать доклад</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RLSpbClose">
          <property name="text">
           <string>Закрыть</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>451</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="RLSgraphicsLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
